### PR TITLE
Update CHANGELOG.md to fix PR number and link

### PR DESCRIPTION
**Pull request by karenzone**
Changelog entry merged in #1195 incorrectly links to #1194 which is still WIP. 
If we go with this approach to make the correction, we'll have to fix it again in release notes for all releases that are locked to this plugin version.

### CODE REVIEW

**Comment by donoghuc** [comment]
Oops! I'm fine with a re-release, not sure if there is another option though.

**Comment by karenzone** [comment]
@donoghuc, I suggest that we approve/merge this PR so that it'll be correct for the next version push. 
Yanking gems can be ugly, and fixing in output isn't the worst. 
If that works for you, it works for me.

**Comment by donoghuc** [comment]
For sure. I realize saying "re-release" is confusing. I didnt mean to try to *replace* the `11.22.10` release. I meant i would be ok with doing a `11.22.11` to trigger the update for the release notes. I guess my question is: do we want to do a release just for the RN update? Or should we just merge this and let it go out in the next release whenever it may go out? 

Also, i just re-kicked the travis run (looks like maybe a race condition in one of our tests), i'll merge this once that goes through green.

**Comment by karenzone** [comment]
> I guess my question is: do we want to do a release just for the RN update? Or should we just merge this and let it go out in the next release whenever it may go out?

No need to bump and re-publish because it wouldn't really change anything in docs anyway.  After this is merged (thank you for offering to handle this, by the way), the changelog will be correct for the next time we update this plugin.

**Review by donoghuc — APPROVED** [review]
Failures just due to no 8.17.1-SNAPSHOT image available yet. this is G2G

**Pull request merged**
